backend/core/serializers.py

```
@@ -1416,22 +1416,95 @@ class TaskTemplateReadSerializer(BaseModelSerializer):
+                # Return evidence IDs only - you can modify this to return more details if needed
+                return [{"id": e.id, "str": e.name} for e in task.evidences.all()]
+            except TaskNode.DoesNotExist:
+                return []
+        return []
+
 
 class TaskTemplateWriteSerializer(BaseModelSerializer):
+    status = serializers.CharField(default="pending")
+    observation = serializers.CharField(allow_null=True, required=False)
+    evidences = serializers.PrimaryKeyRelatedField(
+        many=True, queryset=Evidence.objects.all(), required=False
```

Ruff (0.8.2)

`Evidence` may be undefined, or defined from star imports

(F405)